projects.yaml

```diff
@@ -554,6 +554,13 @@ projects:
   category: datasets
   description: Database of yet-to-be-sythesized materials predicted using state-of-the-art machine learning algorithms
   homepage: https://matterverse.ai/
+- name: Open Databases Integration for Materials Design (OPTIMADE)
+  resource: true
+  category: datasets
+  description: Specification of a common REST API for access to 25+ materials databases
+  homepage: https://optimade.org
```

Comment:
Will comment out `homepage`. CI/CD will pick as URL for the project in the README only one of provided: homepage, docs, repo id, in that order. If the project has a repo, only pick homepage / docs over repo as project URL if the repo does not link to either of them. In most cases, the repo can be considered as root URL, since it links to homepage and docs. This is the case here.